README.md

```diff
@@ -153,10 +153,12 @@ EOF  all  create  destroy  help  quit  show  update
 ## Bugs
 No known bugs at this time. 
 
-## Authors
+## Author
```

Comment:
Typo - fix Authors

Reply:
Oh it's unfortunately I'm going to fix that